pkg/game/board.go

```diff
@@ -358,20 +375,20 @@ func (board *board) updateValidPlacements(tile elements.PlacedTile) {
 	}
 }
 
-func (board *board) checkCompleted(
-	//revive:disable-next-line:unused-parameter Until the TODO is finished.
-	tile elements.PlacedTile,
-) (elements.ScoreReport, error) {
-	// TODO for future tasks:
-	// - identify all completed features
-	// - resolve control of the completed features
-	// - award points
+func (board *board) checkCompleted(tile elements.PlacedTile) elements.ScoreReport {
```

Comment:
Please add comment to this function description, that it removes meeples, but meeples count still needs to be updated using the functions result. It may not obvious, due to moving the functionality here.

Reply:
I added a basic description of the `PlaceTile()` and `addTileToBoard()` methods with a note about this. I do feel it should be obvious since the board has no knowledge of players (it does not encompass them - the `Game` does) but I suppose it doesn't hurt.